No configure options to disable looking for these, redundant for releases
because the tarballs already contain both PDF documentation and man pages,
and as of 2021-05-11 doc generation is not compatible with sphinx-4.0.0+
due to conf.py calling long-deprecated app.add_stylesheet() rather
than app.add_css_file().

--- a/configure.ac
+++ b/configure.ac
@@ -2423,7 +2423,7 @@
     fi
 
 # sphinx for documentation
-    AC_PATH_PROG(HAVE_SPHINXBUILD, sphinx-build, "no")
+    HAVE_SPHINXBUILD="no"
     if test "$HAVE_SPHINXBUILD" = "no"; then
        enable_sphinxbuild=no
        if test -e "$srcdir/doc/userguide/suricata.1"; then
@@ -2434,7 +2434,7 @@
     AM_CONDITIONAL([HAVE_SURICATA_MAN], [test "x$have_suricata_man" = "xyes"])
 
 # pdflatex for the pdf version of the user manual
-    AC_PATH_PROG(HAVE_PDFLATEX, pdflatex, "no")
+    HAVE_PDFLATEX="no"
     if test "$HAVE_PDFLATEX" = "no"; then
        enable_pdflatex=no
     fi
